sequence_processing_pipeline/GenPrepFileJob.py

```diff
@@ -66,11 +67,28 @@ def __init__(self, run_dir, convert_job_path, qc_job_path, output_path,
         # for us. A single call to seqpro will generate n output files, one
         # for each project described in the sample-sheet's Bioinformatics
         # heading.
-        self.command = [self.seqpro_path,
+        self.command = [self.seqpro_path, '--verbose',
                         join(self.output_path, self.run_id),
                         f'"{self.sample_sheet_path}"',
                         join(self.output_path, 'PrepFiles')]
 
+    def _get_prep_file_paths(self, stdout):
+        tmp_l = stdout.split('\n')
+        tmp_l = [x for x in tmp_l if x != '']
+        tmp_d = {}
```

Comment:
Can you use a defaultdict(list) here to reduce the number of lines?

Reply:
ty! I need to remember to use this more.